### PR TITLE
Release 1

### PR DESCRIPTION
**Pull request by hankish**
Release 1 is our first full release of the Quorum1 Governance Repo. What is currently in the repo is "Release 0" which was just a starting point with a single readme file.

Note that the Q-Git Flow that is defined in Release 1 is not yet approved (since we're voting on it now), so we're not using that process. Instead we'll use our standard approval process which is a basic partner vote with a 75% threshold. We'll be conducting this vote both here and in Slack, but will share any Slack context in this thread as well.

## Release 1 Notes

1. **File & Folder Structure:** The basic structure of this repo was defined. The docs are the governance structures themselves, and the flows are the change management processes.

2. **Readme Files:** Each folder has a readme file which is a map to the contents of that folder. There are three readme's for now: Root readme, Flows readme, Docs, readme

3. **Flows:** 
  - **Q-Git Flow:** The Q-Git Flow (QGF) is our base change flow. For now, all of our other flows inherit from this flow and build on top of it, though that may change in the future.
  - **Financial Model Change Flow:** Specifies additional steps required for changing the Financial Model.

4. **Docs:**
  - **Informal Governance Model Definition & Roles:** This doc defines what Quorum1's Informal Governance Processes is, how it relates to our Formal Governance Process, and what key roles are involved.
  - **Financial Model:** The Quorum1 Financial Model defines how we manage our money as a collective. Release 1 contains a direct port of the already approved Financial Model v2 from Notion with a few notable changes. Here is a summary of the process:
    - The "Financial Model v2" document was exported as a markdown file from our Notion instance and imported into the repo exactly as is.
    - The "Table of Contents" section was removed, since it didn't contain come across in the export and a table of contents feature is built in to Github.
    - All of the links within the doc were edited so that they properly point to the appropriate section, using Github's standard header-based linking syntax. One header had to be changed in order to make it unique. Note: There were no links pointing to anything outside of the doc itself (all of the links were and still are self-contained).
    - A couple of small formatting changes were made to fix the appearance of the math formulas in one part of the doc.
    - All named individuals were either removed or replaced with their Quorum1 member numbers. 
    - One named company was replaced with a more generic term (initial legal counsel).

### CODE REVIEW

File: docs/igm-roles.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+_**Dependencies:** None_
+
+This doc defines what Quorum1's Informal Governance Processes is, how it relates to our Formal Governance Process, and what key roles are involved.
+
+## Note on Informal Governance Model
+
+Quorum1 is currently operating in our "Informal Governance Model" (IGM). 
+
+There is a deadline to adopt our "Formal Governance Model" (FGM) by December 31, 2025. The FGM will be a major release building on the ideation and design work which has been happening since before Quorum1's founding. FGM will be highly participatory, highly stigmergic, and designed to facilitate the continuing innovation and evolution of Quorum1 well into the future.
```
Comment (yvettie86) [inline docs/igm-roles.md:11]: Can we please reconsider the acronym "FGM"?

**Review by yvettie86 — APPROVED** [review]
Have read through the docs. My only comment isn't a showstopper, so am approving!

**Review by edjieun — APPROVED** [review]
I have read and reviewed Release-1 Pull #2 , and approve on behalf of Ed Hwang and Stephanie Caulley.

File: docs/igm-roles.md (hunk 1 above)
Comment (hankish) [reply docs/igm-roles.md:11]: We've already got legal docs which use that acronym... but yes let's tweak this in the next release.

**Comment by hankish** [comment]
I'm approving on behalf of myself, Jay, & Trak.

**Comment by hankish** [comment]
I'm approving on behalf of myself, Jay, Prabhakar, & Trak.

**Comment by hankish** [comment]
Posting the screenshots from the slack voting thread here for transparency. 

Slack Screenshot (1 of 2) - 2024-07-03 at 1 36 38 PM
Slack Screenshot (2 of 2) - 2024-07-03 at 1 37 01 PM

**Comment by hankish** [comment]
The vote is complete. Summary below.

**Official Vote Update:** We have 7 approvals out of 9, so this vote passes! 

Here is a summary of the approvals:
- Ed approved on behalf of himself and Stephanie
- Hank approved on behalf of himself, Prabhakar, Jay, and Trak
- Yvette approved for herself
- No response from Jon or Esteban

Thank you everyone!

**Pull request merged**